worker.go

```diff
@@ -159,7 +171,7 @@ func go_frankenphp_worker_handle_request_start(mrh C.uintptr_t) C.uintptr_t {
 
 	var r *http.Request
 	select {
-	case <-done:
```

Comment:
Shouldn't this be preserved too?

Reply:
It's not necessary since `workersDone` will always close before done is closed

frankenphp.go `Shutdown()` 
worker.go `stopWorkers()` 
worker.go `close(workersDone)`
frankenphp.go `close(done)`

It would only be necessary if `done` was closed from somewhere else as well